common_utils/audit_logging.py

```diff
@@ -36,39 +61,61 @@ def log(action, instance):
     message = {
         "audit_event": {
             "origin": "JASSARI-BE",
+            "operation": action,
             "status": "SUCCESS",
             "date_time_epoch": int(current_time.timestamp()),
-            "date_time": f"{current_time.isoformat(sep='T', timespec='milliseconds')}Z",
+            "date_time": f"{current_time.replace(tzinfo=None).isoformat(sep='T', timespec='milliseconds')}Z",
             "actor": {"role": _resolve_role(current_user, profile)},
             "actor_service": {
                 "id": "youth_membership",
                 "name": "Youth Membership",
             },
-            "operation": action,
             "target": {
                 "profile_id": profile_id,
                 "profile_part": instance.__class__.__name__,
             },
         }
     }
 
-    if current_user:
-        message["audit_event"]["actor"]["user_id"] = (
-            str(current_user.uuid) if hasattr(current_user, "uuid") else None
-        )
-        if settings.AUDIT_LOG_USERNAME:
-            message["audit_event"]["actor"]["user_name"] = (
-                current_user.username if hasattr(current_user, "username") else None
-            )
+    _format_user_data(message["audit_event"], "actor", current_user)
+    _format_user_data(message["audit_event"], "target", target_user)
 
-    if target_user:
-        message["audit_event"]["target"]["user_id"] = (
-            str(target_user.uuid) if hasattr(target_user, "uuid") else None
-        )
-        if settings.AUDIT_LOG_USERNAME:
-            message["audit_event"]["target"]["user_name"] = (
-                target_user.username if hasattr(target_user, "username") else None
-            )
+    ip_address = get_original_client_ip()
+    if ip_address:
+        message["audit_event"]["jassaribe"] = {
```

Comment:
What's the reasoning behind naming the extra field as `jassaribe`? Would this mean that if Berth did something similar with their logging, they would add a new extra field as `venebe`? If an admin would want to look for all log entries done from the same IP address, they would have to make sure to include all possible backends to their search, right?

Would this be better if we named it as something more generic? Like `extra_info` or something?

Reply:
I basically just mimicked what was done in open-city-profile where this field was `profilebe`, but someting like `extra_info` does sound quite reasonable. :+1: